objects/buttons.cpp

```diff
@@ -0,0 +1,37 @@
+class RebootButton
+{
+private:
+    Texture rebootButton;
+
+public:
+    void loadAssets()
+    {
+        rebootButton.loadFromFile("assets/sprites/rebootBtn.png");
+        assets["rebootButton"] = rebootButton;
+    }
+
+    void setSprites()
+    {
+        sprites["rebootButton"].setTexture(assets["rebootButton"]);
+        sprites["rebootButton"].setPosition(261, 1); // Need to change, but don't know how :)
```

Comment:
Suggestion: 
`void setSprite(int x, int y = 1)`

Class knows which sprite it needs to use - it is okay.
But class should not have hardcoded coordinates because some games needs button on `x=261, y=1` and some games needs button on `x=61, y=1`
You don't need to build class only for 1 game (use case). You need to create universal class, that can fit for every game but with different params